### ISSUE TITLE
[Compile]: Could not determine the dependencies of task ':connectivity_plus:compileReleaseJavaWithJavac'.

### ISSUE THREAD

**Issue by naqibam** [fluttercommunity/plus_plugins#3353]
### Operating System

Windows 11

### Plugin

connectivity_plus

### Version

6.1.0

### Flutter SDK

3.24.4

### Compilation command

flutter run

### Dart Dependencies

```yaml
dependencies:
  hand_signature: ^3.0.2
  dio: ^5.4.0
  pdf: ^3.10.7
  file_picker: ^8.0.0+1
  flutter:
    sdk: flutter
  flutter_dotenv: ^5.1.0
  flutter_localization: ^0.2.0
  flutter_secure_storage: ^9.2.2
  go_router: ^13.0.1
  google_fonts: ^6.1.0
  image_picker: ^1.0.4
  intl: ^0.18.1
  json_annotation: ^4.8.1
  omni_datetime_picker: ^1.0.9
  path: ^1.8.3
  provider: ^6.1.1
  responsive_framework: ^1.1.1
  signature: ^5.4.0
  skeletonizer: ^1.0.1
  sqflite_sqlcipher: ^2.2.1
  open_file: ^3.3.2
  path_provider: ^2.1.2
  syncfusion_flutter_pdf: ^27.1.52
  firebase_core: ^3.1.1
  firebase_messaging: ^15.0.2
  unique_identifier: ^0.3.0
  gap: ^3.0.1
  mime: ^1.0.5
  intl_phone_field: ^3.2.0
  flutter_animate: ^4.5.0
  flutter_inset_box_shadow: ^1.0.8
  dotted_border: ^2.1.0
  uuid: ^4.3.3
  synchronized: ^3.1.0+1
  fs_shim: ^2.3.0+1
  path_provider_platform_interface: ^2.1.2
  dropdown_search: ^5.0.6
  http: ^1.2.1
  pretty_dio_logger: ^1.3.1
  flutter_local_notifications: ^17.2.3
  pretty_qr_code: ^3.3.0
  easy_pdf_viewer: ^1.0.8
  http_parser: ^4.0.2
  syncfusion_flutter_pdfviewer: ^27.1.52 
  package_info_plus: ^8.0.0
  photo_view: ^0.15.0
  workmanager: ^0.5.2
  flutter_bcrypt: ^1.0.8
  encrypt: ^5.0.3
  internet_connection_checker_plus: ^2.4.1
  permission_handler: ^11.3.1
  firebase_crashlytics: ^4.0.2
  pin_code_fields: ^8.0.1
  flutter_timezone: ^2.0.1
  flutter_screenutil: ^5.9.3
  printing: ^5.7.2
  pdf_manipulator: ^0.5.9
  open_file_manager: ^1.0.2
  flutter_date_range_picker: ^0.1.1
  flutter_windowmanager: ^0.2.0
  ssl_pinning_plugin: ^2.0.0
  safe_device: ^1.1.8
  device_info_plus: ^10.1.2
  win32: ^5.5.5

dependency_overrides:
  intl: ^0.18.1


dev_dependencies:
  flutter_test:
    sdk: flutter

  flutter_lints: ^3.0.1
  json_serializable: ^6.7.1
  build_runner: ^2.4.7
  change_app_package_name: ^1.1.0
```

### Compilation Error

```shell
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':connectivity_plus:compileReleaseJavaWithJavac'.
> Cannot query the value of this provider because it has no value available.
```

### Flutter Doctor

```shell
[√] Flutter (Channel stable, 3.24.4, on Microsoft Windows [Version 10.0.26100.2033], locale en-SG)
    • Flutter version 3.24.4 on channel stable at C:\VSC\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 603104015d (2 weeks ago), 2024-10-24 08:01:25 -0700
    • Engine revision db49896cf2
    • Dart version 3.5.4
    • DevTools version 2.37.3

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    • Android SDK at C:\Users\Name\AppData\Local\Android\sdk
    • Platform android-35, build-tools 35.0.0
    • Java binary at: C:/Program Files/Java/jdk-18.0.2.1\bin\java
    • Java version Java(TM) SE Runtime Environment (build 18.0.2.1+1-1)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop Windows apps
    X Visual Studio not installed; this is necessary to develop Windows apps.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2024.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.11+0--11852314)

[√] VS Code (version 1.95.2)
    • VS Code at C:\Users\Name\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.100.0

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5556 • android-x64    • Android 15 (API 35) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.26100.2033]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 130.0.6723.117
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 130.0.2849.68

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

### Checklist before submitting a bug

- [X] I searched issues in this repository and couldn't find such bug/problem
- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I'm using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

**Comment by Webdevelopertr2021** [comment]
any news i have same problem :(

**Comment by vbuberen** [comment]
Have you checked https://github.com/fluttercommunity/plus_plugins/issues/3303 and https://github.com/fluttercommunity/plus_plugins/issues/3299 and can confirm that your project satisfies the requirements, especially Gradle plugin and wrapper versions?

**Comment by naqibam** [comment]
> Have you checked #3303 and #3299 and can confirm that your project satisfies the requirements, especially Gradle plugin and wrapper versions?

These issues shouldn't apply since I intentionally downgraded Android studio to a version before Ladybug to counter this problem. My gradle version in the distributionUrl was 7.6.

**Comment by vbuberen** [comment]
Alright then you had Gradle wrapper version lower than required by the plugin.
After changing to the required version the issue is gone now for you?

**Comment by miquelbeltran** [comment]
Pasting the same messaged I posted in #3299 

I'll lock this conversation as it has been resolved. If you are still struggling with having your Android project to build, check the meta-ticket #3303 or look on Stack Overflow for solutions.

Gradle compilation issues and incompatibility of Flutter projects with Android Studio and JDK 21 are not a plus_plugins issue, rather, it should be resolved by the developer or reported to the Flutter team.
